qemu/tests/cfg/fio_perf.cfg

```diff
@@ -0,0 +1,78 @@
+    threads = "16"
+    rw = "read write randread randwrite randrw"
+    variants:
+        - localfs:
+            image_name_disk1 = /mnt/storage2
+        - nfs_block:
+            image_name_disk1 = images/storage2
+        - fusion-io:
+            image_name_disk1 = /dev/fioa
+    variants:
+        - file_system:
+            image_size_disk1 = 40G
+            force_create_image_disk1 = yes
+            remove_image_disk1 = yes
+            format = True
+        - raw_disk:
+            image_raw_device_disk1 = yes
+            vhost_nic1 =
+    variants:
+        - single_disk:
+            images += " disk1"
+
+    variants:
+        - fio_linux:
+            only Linux
+            pattern = ".*[read|write].*bw=(\d+(?:\.\d+)?[\w|\s]B/s),\siops=(\d+).*"
+            order_list = "Block_size Iodepth Threads BW(MB/S) IOPS Latency(ms) Host_CPU BW/CPU KVM_Exits Util%"
+            guest_ver_cmd = "uname -r"
+            compile_cmd = "make && make install"
+            pre_cmd = "i=`/bin/ls /dev/[vs]db` && mkfs.xfs $i > /dev/null; partprobe; umount /mnt; mount $i /mnt"
+            drop_cache = "sync && echo 3 > /proc/sys/vm/drop_caches"
+            guest_result_file = /tmp/fio_result
+            variants:
+                - dedicated_mach1:
+                    mac_nic1 = 9a:37:37:37:37:4e
+                    fio_cmd = "fio --rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=/mnt/%s --name=job1 --ioengine=libaio --thread --group_reporting --numjobs=%s --size=512MB --time_based --output=/tmp/fio_result &> /dev/null"
+                    vhost_nic1 =
+                    iothreads = iothread0
+                    virtio_blk:
+                        blk_extra_params_image1 = "iothread=${iothreads}"
+                    virtio_scsi:
+                        bus_extra_params_image1 = "iothread=${iothreads}"
+                - dedicated_mach2:
+                    mac_nic1 = 9a:37:37:37:37:5e
+                    fio_cmd = "fio --rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=/mnt/%s --name=job1 --ioengine=libaio --thread --group_reporting --numjobs=%s --size=512MB --time_based --output=/tmp/fio_result &> /dev/null"
+                - dedicated_mach3:
+                    mac_nic1 = 9a:37:37:37:37:7e
+                    #fio_cmd = "/bin/ls /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_virtio-scsi2-id0 && i=`python -c "import os; print os.path.realpath('/dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_virtio-scsi2-id0')"` && fio --rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=$i --name=job1 --ioengine=libaio --thread --group_reporting --numjobs=%s --time_based --output=/tmp/fio_result &> /dev/null"
+                    fio_cmd = "i=`/bin/ls /dev/[vs]db` && fio --rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=$i --name=job1 --ioengine=libaio --thread --group_reporting --numjobs=%s --time_based --output=/tmp/fio_result &> /dev/null"
+                    image_aio = native
+        - fio_win:
```

Comment:
You mean fio_windows.py and fio_windows.cfg? This is just a simple function test, running fio in guest. It does not meet our performance test requirement, e.g. we need test different IO patterns, block sizes, iodepths... and we need some special envs, e.g. numa pin, specify dada disk, format data disk , free memory...